src/main/java/rsb/internal/globval/GlobalConfiguration.java

```diff
@@ -47,27 +47,38 @@ public static String getAccountsFile() {
 			if (GlobalConfiguration.getCurrentOperatingSystem() == OperatingSystem.WINDOWS) {
 				path = System.getenv("APPDATA") + File.separator
 						+ GlobalConfiguration.NAME + "_Accounts.ini";
+			} else if (GlobalConfiguration.getCurrentOperatingSystem() == OperatingSystem.LINUX) {
+				path = Paths.getUnixHome() + File.separator
+						+ ".config" + File.separator
+						+ GlobalConfiguration.NAME_LOWERCASE + "_acct.ini";
 			} else {
 				path = Paths.getUnixHome() + File.separator + "."
 						+ GlobalConfiguration.NAME_LOWERCASE + "acct";
 			}
 			return path;
 		}
 
-		public static String getHomeDirectory() {
+		public static String GetOsrsBotDirectory() {
```

Comment:
Renamed method to be more useful

Flipped if statement to make smaller branch on top

Made Linux directory go inside .config again

Reply:
Java methods in the repo should be camelcase with a lowercase initial letter. 
Not a big deal.

Reply:
Whoops used to C#